Added dataset appendix that is used in I2ML + SL

xuansontrinh opened the pull request:
The following dataset appendix is included in this PR:

- [x] boston
- [x] circle
- [x] credit
- [x] glass
- [x] ionosphere
- [x] iris
- [x] mnist
- [x] sonar
- [x] spam
- [x] spirals
- [x] titanic
- [x] waveform

xuansontrinh commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Generate dataset appendix](https://trello.com/c/WaeRWfYU/208-generate-dataset-appendix)

ludwigbothmann commented on datasets/slides/boston.rmd:
```diff
@@ -0,0 +1,144 @@
+---
```
Please include the discussion of potential ethical problems for the Boston housing dataset, see, e.g., https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_boston.html

ludwigbothmann commented on datasets/slides/credit.rmd:
````diff
@@ -0,0 +1,167 @@
+theme_set(theme_bw())
+```
+
+German Credit Dataset is a research dataset from the University of Hamburg from 1994 and donated by Prof. Hans Hoffman. Each entry represents a person who takes a credit by a bank. Each person is classified as "good" or "bad" credit risks according to the set of attributes.
````
Please add a link to the original source / paper

ludwigbothmann commented on datasets/slides/glass.rmd:
```diff
@@ -0,0 +1,147 @@
+
+The glass identification dataset was created by B. German, Central Research Establishment, Home Office Forensic Science Service, Aldermaston, Reading, Berkshire RG7 4PNA. The data frame with 214 observations has examples of the chemical analysis of 7 varieties of glass. The target is to predict the kind of glass type based on the chemical analysis.
+
+(ref:glass-cap) Source: macrovector ([link](https://www.freepik.com/free-vector/empty-realistic-glasses-set-different-alcohol-drinks-cocktails-isolated_10347034.htm#query=glass&position=4&from_view=search))
```
this image doesn't really represent the target label, does it?

xuansontrinh replied:
No, this image is just for a nice visualization, like the Boston images in the Boston dataset.

ludwigbothmann commented on datasets/slides/ionosphere.rmd:
````diff
@@ -0,0 +1,126 @@
+theme_set(theme_bw())
+```
+
+This radar data was collected by a system near Goose Bay, Labrador. This system is made up of a phased array of 16 high-frequency antennas with a total transmission power of around 6.4 kilowatts. The free electrons in the ionosphere were the targets. "Good" radar returns are those that show indications of ionosphere structure. Those with "poor" returns do not; their signals pass through the ionos ...
````
Is there a citable source?

ludwigbothmann reviewed (commented):
Where possible, please cite original sources.

xuansontrinh commented:
Thank you @ludwigbothmann for your comments, I will address them later.
Is it okay if I add our course name to the header of the appendix?
And if it is, what will be the text that you want to show up in the header?

xuansontrinh commented:
Hello @ludwigbothmann,
I addressed your comments in my latest commit + added headers for the appendix.
Please take a look to see if it is good now.
Thank you very much!
